bazaarvoice/emodb
# fix local docker stash, megabus roles, remove unused blob role

## Github Issue #

-

## What Are We Doing Here?

- fix local docker megabus and stash roles
- remove unused blob role

## How to Test and Verify

1. Check out this PR
2. Follow instructions described in https://github.com/bazaarvoice/emodb/blob/fix_docker/docker/README.md
3. Run emodb-web, megabus and stash via docker compose
4. Profit

## Risk

### Level 

`Low`

### Required Testing

`Smoke`

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
